react/components/FilterSidebar.js

```diff
@@ -71,6 +73,14 @@ const FilterSidebar = ({
   // eslint-disable-next-line @typescript-eslint/no-use-before-define
   const currentTree = useCategoryTree(tree, categoryTreeOperations)
 
+  const selectedFacets = React.useMemo(() => { 
+    const filterKeys = filters.map(filter => filter.key);
+
+    return selectedFilters.filter((facet) => (
+      filterKeys.includes(facet.key) 
+      && facet.selected
+    )) }, [filters, selectedFilters]);
```

Comment:
why do we need to do this? I think `selectedFacets` will always be the same thing as `selectedFilters` 🤔

Reply:
@thalytafabrine thanks for the reply!

In some cases, `selectedFilters` has categories included. In [this page](https://adicionabadgenosfiltrosselecionados--oficinareserva.myvtex.com/roupas/polos/m/preto?initialMap=c,c&initialQuery=roupas/polos&map=category-1,category-2,tamanho,cor), for instance, we have "Roupas" and "Polo" as selected filters. 

This `useMemo` extracts from `selectedFilters` only filters that are shown in the side menu.

![Screenshot from 2022-02-16 17-45-04](https://user-images.githubusercontent.com/15324745/154353950-efcb4050-8110-4e91-9c8a-b981a03d678e.png)